commit 83888db91de8a8adaf29a4fd123f49448c86abaf
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Sat Jul 30 19:02:44 2022 +0900

    npm set-script start "ts-node-dev --respawn src/index.ts"

diff --git a/app/package.json b/app/package.json
index 3f203d9..696f207 100644
--- a/app/package.json
+++ b/app/package.json
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node-dev --respawn src/index.ts"
   },
   "keywords": [],
   "author": "",
